thrift/messaging.thrift

```diff
@@ -154,13 +155,12 @@ union VerificationRecord {
     4: Phase_4_msg p4
 }
 
-union Phase_5_msg {
-    1: Transaction transaction,
-    2: VerificationRecord verification_record,
-    /* Level 5 node will NOT hash this field */
-    3: string hash,
-    /* Level 5 node WILL hash this field */
-    4: string misc
+union Phase_5_request {
+    1: VerificationRecord verification_record
+}
+
+struct Phase_5_msg { /* change to Public_5_msg */
```

Comment:
Is `Phase_5_msg` still used?
I see a reference in the `gen` code, but I thought it was replaced by `Phase_5_request`?